src/modules/EdgeDetect/EdgeUtils.js

```diff
@@ -98,11 +98,11 @@ function categorizeAngle(angle){
   */
 }
 
-function isOutOfBounds(pixels, x, y){
+function isOutOfBounds(pixels, x, y){ // checks if a pixel is outside the bounds of the image. Error handling for convolution
   return ((x < 0) || (y < 0) || (x >= pixels.shape[0]) || (y >= pixels.shape[1]));
 }
 
-const removeElem = (arr = [], elem) => {
+const removeElem = (arr = [], elem) => { // removes an element from an array
```

Comment:
which element is removed?
I told you comments should be unambiguous.

Reply:
I'll make it jsdoc then